R/stan_betareg.fit.R

```diff
@@ -24,7 +24,8 @@ stan_betareg.fit <- function(x, y, z = NULL,
                              link.phi = NULL, ...,
                              prior = normal(), prior_intercept = normal(),
                              prior_z = normal(), prior_intercept_z = normal(),
-                             prior_ops = prior_options(), prior_PD = FALSE, 
+                             prior_dispersion = cauchy(0, 5),
+                             prior_ops = NULL, prior_PD = FALSE, 
```

Comment:
Are you referring to `prior_ops` not being user defined or not being in `stan_betareg.fit` at all. If I pull out prior_ops = NULL then I have to pull it out of `prior_info` here https://github.com/stan-dev/rstanarm/blob/feature/betareg/R/stan_betareg.fit.R#L216 and from `summarize_betareg_prior` here https://github.com/stan-dev/rstanarm/blob/feature/betareg/R/stan_betareg.fit.R#L302 which causes some of your prior_summary stuff to break.